src/test/kotlin/lotto/domain/LottoFactoryTest.kt

```diff
@@ -41,7 +43,20 @@ class LottoFactoryTest : StringSpec({
         val lottoList = lottoFactory.createLottoList()
 
         // Then
-        lottoList.forEach { lotto -> lotto.value shouldBe expectedLottoNumbers }
+        lottoList.forEach { lotto -> lotto.value shouldBe expectedLottoNumbers.map { LottoNumber(it) } }
+    }
+
+    "{given} 정수 리스트, size=6, {when} LottoFactory.fromList() {then} Lotto 생성" {
+        val list = listOf(1, 2, 3, 4, 5, 6)
+        val result = LottoFactory.fromList(list)
+        // assertTrue(result is Lotto) FIXME : this check is useless ?
```

Comment:
```suggestion
        assertThat(result.value).containsExactlyElementsOf(list.map { LottoNumber(it) })
```

이런식으로 내가 의도한 LottoNumber객체로 잘 변환해서 만들어졌는지 확인해볼 수 있을 것 같아요!

Reply:
containsXXX 이런 함수 사용해보면 좋을거같네요. 감사합니다!